.travis.yml

```diff
@@ -1,17 +1,42 @@
 sudo: false
 
+script: bundle exec rake
+
 services:
   - mongodb
   - memcache
 
 rvm:
-  - 1.9.3
   - 2.0.0
-  - 2.2.0
+  - 2.1.10
+  - 2.2.10
+  - 2.3.7
+  - 2.4.6
+  - 2.5.5
+  - 2.6.3
 
 env:
   - MONGOID_VERSION=3
   - MONGOID_VERSION=4
   - MONGOID_VERSION=5
+  - MONGOID_VERSION=6
+  - MONGOID_VERSION=7
 
 cache: bundler
+
+matrix:
+  include:
+    - rvm: 2.6.3
+      env: RUBOCOP=true
+      script: bundle exec rake rubocop
+  exclude:
+    - rvm: 2.0.0
+      env: MONGOID_VERSION=6
+    - rvm: 2.1.10
+      env: MONGOID_VERSION=6
+    - rvm: 2.0.0
+      env: MONGOID_VERSION=7
+    - rvm: 2.1.10
+      env: MONGOID_VERSION=7
+  allow_failures:
+    - env: MONGOID_VERSION=7
```

Comment:
Just to be clear, I couldn't get the builds for Mongoid 7 so leaving this here for now, but I would like to get them pass once I have enough time to fix them.

Reply:
Ok let's deal with that next.